### PR TITLE
Add ARM build support 

### PR DESCRIPTION
**Pull request by mdh34**
In reference to #91

### CODE REVIEW

**Comment by mdh34** [comment]
Fixed permissions as requested :)

File: src/infer/inferOs.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function inferPlatform() {
 
 function inferArch() {
   const arch = os.arch();
-  if (arch !== 'ia32' && arch !== 'x64') {
+  if (arch !== 'ia32' && arch !== 'x64' && arch !== 'arm') {
```
Comment (ronjouch) [inline src/infer/inferOs.js]: @mdh34 so, the command-line flag is `armv7l`, but here `os.arch()` returns just `arm`? (Just checking, if yes it's alright).

Comment (mdh34) [inline src/infer/inferOs.js]: I believe, yes - `os.arch()` only returns `arm` or `arm64`, as per [https://nodejs.org/api/os.html](url)

Comment (ronjouch) [inline src/infer/inferOs.js]: @mdh34 👍, great then. Merging, thanks again for the contribution! Out of curiosity, on what kind of device do you use Nativefier? Raspberry pi?

Comment (mdh34) [inline src/infer/inferOs.js]: @ronjouch Mostly just on desktop, but I can see this being super useful for projects on devices like the pi

Comment (FredericGuilbault) [reply src/infer/inferOs.js]: @ronjouch Im the maintainer of lysmarineOS and I looking to use it to display the local run of the signalK server. and other web based local services as freeboard-sk pypilot, maybe some sensors/stat display.

**Pull request merged**